### PR TITLE
Fix PointListReader for reading single-element (Point1D case) per line

### PR DESCRIPTION
**Pull request by Seangongx**
# PR Description

* Fixed the failure when give the extreme input case by single-element(Like `typedef DGtal::PointVector<1, DGtal::int32_t> Point1D`) using getPolygonsFromInputStream() function.
* Changed the sample data and added new test.

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug mode.
- [x] All continuous integration tests pass (Github Actions)

### CODE REVIEW

File: tests/io/readers/testPointListReader.cpp [hunk 1]
```diff
@@ -91,9 +91,29 @@ bool testPointListReader()
     }
     trace.info()<< std::endl;
   }
-  nbok += (vectPolygons.at(0).size()==2 && vectPolygons.at(1).size()==2 && vectPolygons.at(0).at(0)==Z3i::Point(1,2,3)
-           && vectPolygons.at(0).at(1)==Z3i::Point(4,5,6) && vectPolygons.at(1).at(0)==Z3i::Point(0,0,10)
-           && vectPolygons.at(1).at(1)==Z3i::Point(0,5,5)) ? 1 : 0; 
+  trace.beginBlock ( "Testing reading spectial 1 dimension elements (several by lines)..." );
```
Comment (kerautret) [inline tests/io/readers/testPointListReader.cpp:94]:
```suggestion
  trace.beginBlock ( "Testing reading special 1 dimension elements (polygon of 1 dimension point)..." );
```

**Review by kerautret — CHANGES_REQUESTED** [review]
Thanks @Seangongx just a typo and could you please add an entry on the changelog file that probably forget to add ?

File: tests/io/readers/testPointListReader.cpp (hunk 1 above)
Comment (Seangongx) [reply tests/io/readers/testPointListReader.cpp:94]: Update done

Comment (kerautret) [reply tests/io/readers/testPointListReader.cpp:94]: you could just click in commit suggestion given above ;)

Comment (kerautret) [reply tests/io/readers/testPointListReader.cpp:94]: @Seangongx I think you miss it

**Review by kerautret — APPROVED** [review]
Fine for me thanks @Seangongx

**Pull request merged**